components/about/KofiWidget.vue

```diff
@@ -0,0 +1,53 @@
+<template>  
```

Comment:
ty for grouping the components! we've finally been creating too many for a flat tree

Reply:
Yeah it turns out that the Vue / Nuxt system for this is fairly easy to reason with!